tests/test_impersonated_credentials.py

```diff
@@ -473,6 +473,7 @@ def test_id_token_from_credential(
 
         assert id_creds.token == ID_TOKEN_DATA
         assert id_creds._include_email is True
+        assert id_creds._target_credentials is credentials
```

Comment:
the same creds are used in ln 469 and 471. So there is no point of making the `from_credentials` call in 471 I think?

Reply:
Updated with a fix for that